pkg/remote/test/github_teams_multiple/github_team-4556811.res.golden.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You should remove these test files from supplier directory

Reply:
sorry removed them in the wrong pr when I comitted :sweat: